### ISSUE TITLE
Decreased ferritin concentration

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6618]
**Preferred term label:** Decreased ferritin concentration

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
Abnormal ferritin concentration (once created)

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
@eswietlik These terms already exist, see https://hpo.jax.org/app/browse/term/HP:0040133
